### PR TITLE
feat(switch): initial implementation of the switch component

### PR DESCRIPTION
**Pull request by ggonchar**
### Description

The initial Switch component implementation according to the [design](https://www.figma.com/file/rrvkvQEoVTOHa7MzyKSaoz/OTP-Global-Design-System?node-id=1370%3A40767). I left disabled state out of scope as it's not used yet anywhere across our web applications.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation change

### Did you remember to take care of the following?

- [x] `npm i` – for new NPM dependencies.
- [x] `npm run lint` - to check for linting issues
- [x] `npm run test` - to run unit tests
- [x] `npm run test:screenshots` - to run snapshot tests

### New Feature / Bug Fix

- [x] Run unit tests to ensure all existing tests are still passing.
- [ ] Add new passing unit tests to cover the code introduced by your pr.

Thanks for contributing!

### CODE REVIEW

**Review by derek-heycar — CHANGES_REQUESTED** [review]
`packages/switch/README.md` should contain a brief description, short information about installation, usage, and a link to the storybook, please check others package.

File: packages/switch/src/Switch.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  onChange,
+}) => {
+  const handleChange = useCallback(
+    event => onChange?.(event.target.checked),
+    [onChange],
+  );
+
+  return (
+    <label className={styles.container}>
+      <input checked={checked} onChange={handleChange} type="checkbox" />
```
Comment (derek-heycar) [inline packages/switch/src/Switch.tsx:19]: just 

```
onChange={onChange}
```

File: packages/switch/src/Switch.types.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export interface SwitchProps {
+  /**
+   * If true, the switch is displayed as checked
+   */
+  checked?: boolean;
+  /**
+   * Function that will be called every time the Switch value is changed. The argument `isChecked` represent the new/target value of the `checked` attribute.
+   */
+  onChange?: (isChecked: boolean) => void;
```
Comment (derek-heycar) [inline packages/switch/src/Switch.types.ts:9]: what about `disabled` state and the `className` prop ?

File: packages/switch/src/Switch.tsx (hunk 1 above)
Comment (derek-heycar) [inline packages/switch/src/Switch.tsx:19]: do not forget about ARIA attributes 
```
        role="switch"
        aria-checked=""
```

**Review by derek-heycar — CHANGES_REQUESTED** [review]
Remove please from the sandbox `onChange` control, it is useless.

File: packages/switch/src/Switch.types.ts (hunk 2 above)
Comment (ggonchar) [reply packages/switch/src/Switch.types.ts]: `className` prop added. `disabled` state will add in a separate PR https://heycar.atlassian.net/browse/HEYUI-119

Comment (ggonchar) [reply packages/switch/src/Switch.types.ts]: Actually, added `disabled` state to this PR as well

**Pull request merged**